Welthungerhilfe/cgm-ml
# Angle calculation hotfix (radial instead of linear)

# Description

In triangle ABC  (where |BC| == 1, A is position of the camera and B is the end of the forward vector)
A--B
|   /
C
The angle is not linearly dependend on  |AB|/|BC|, correct is asin(|AB|/|BC|}

# How Has This Been Tested?

* run CV/evaluation.py
* sort output.py by the angle (using LibreOffice)
* run CV/renderer.py
* check the visualisations are ordered by angle

Maximal value in raw dataset: -10° (berore it was -16°):
![1](https://user-images.githubusercontent.com/6472545/129679724-00d2f8eb-8d01-47f3-a44b-c0895921d78a.png)
Minimal value in raw dataset: -78° (before it was -87°):
![670](https://user-images.githubusercontent.com/6472545/129679748-ff47a1fb-00a6-4fe3-b5eb-77d1d73d854d.png)

Checking the floor perspective on the second image, this image is not close to -90°
